cirrus/server/cirrus/experiment_recipes.py

```diff
@@ -18,38 +17,38 @@ class RecipeType(Enum):
 
 
 class RemoteSettings:
-    def __init__(self, sdk: SDK):
+    def __init__(self, url: str, sdk: SDK):
         self.recipes: dict[str, list[Any]] = {"data": []}
-        self.url: str = remote_setting_url
+        self.url: str = url
         self.sdk = sdk
 
     def get_recipes(self) -> dict[str, list[Any]]:
         return self.recipes
 
     def get_recipe_type(self, experiment_slug: str) -> str:
-        for experiment in self.get_recipes()["data"]:
+        for experiment in self.recipes["data"]:
             if experiment["slug"] == experiment_slug:
-                if experiment.get("isRollout", False):
-                    return RecipeType.ROLLOUT.value
-                else:
-                    return RecipeType.EXPERIMENT.value
-
+                return (
+                    RecipeType.ROLLOUT.value
+                    if experiment.get("isRollout", False)
+                    else RecipeType.EXPERIMENT.value
+                )
```

Comment:
If you're trying to find the recipe out of the list in recipes you can also just do it like

```py
experiment = {r['slug']: r for r in self.recipes['data']}.get(experiment_slug)
if experiment is not None:
    return ....
```